src/lib.rs

```diff
@@ -7,7 +7,8 @@
 /// # Examples
 ///
 /// ```should panic
-/// # #[macro_use] extern crate assert_approx_eq;
+/// #[macro_use] 
+///  extern crate assert_approx_eq;
```

Comment:
sorry to be nitpicky but it appears there is a space here! mind removing that as well?